core/google/cloud/future/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017, Google Inc.
```

Comment:
Yes, my next PR will drop an `operation.py` module in here for `google.cloud.future.operation` unless you would prefer me to replace the current `google.cloud.operation` module.